app/services/suggester/users/sidebar.rb

```diff
@@ -4,33 +4,39 @@ class Sidebar
       def initialize(user, given_tag)
         @user = user
         @given_tag = given_tag
+        @minimum_reaction_count = Rails.env.production? ? 25 : 0
```

Comment:
what if we just keep it to 25? is there a reason why in development it should be zero?

Reply:
With a minimum of 25 reactions, I don't think any local dev user will meet that threshold from the get-go. With 0, it will ensure we will always see suggested user when browsing to `localhost:3000/t/anytagpage`. 

Yeah I agree this code doesn't look very peachy to me either and I'm okay with keeping it locked at 25 but I also don't want us dev to run into confusion when developing. What do you think?

Reply:
let's keep it!